doc/python/recipe.rst

```diff
@@ -72,10 +72,10 @@ Recipe
 
     .. function:: probes(gid)
 
-        Returns a list specifying the probe addresses describing probes on the cell ``gid``.
-        Each address in the list is an opaque object of type :class:`probe` produced by
-        cell kind-specific probe address functions. Each probe address in the list
-        has a corresponding probe id of type :class:`cell_member`: an id ``(gid, i)``
+        Returns a list specifying the probesets describing probes on the cell ``gid``.
+        Each element in the list is an opaque object of type :class:`probe` produced by
```

Comment:
It's the Python version of cpps `probe_info`, which is what probes must return. So, I guess it's a bit of an internal interface, which is why it's not documented yet (nor has a useful docstring). I actually also don't know exactly how you'd make your own probes, but I suppose it will always need to return a meaningful `probe_info`/`probe`. I'll add a definition with a hint to `probe_info`. In future we can round it out if need be.

Reply:
> Making your own probes 

As in 'add a new quantity to be probed'? You'll need to go down into the C++ code and recompile.

Reply:
How about adding 
> `probe`: opaque handle, unused in Python as of now.

Reply:
I see this was done above, nice.